const asyncErrorHandler = (handler) => {
  return async (req, res, next) => {
    try {
      await handler(req, res);
    } catch (err) {
      //log the err to db∆í
      next(err);
    }
  };
};

module.exports = asyncErrorHandler;
